lib/negroku/tasks/eye.rake

```diff
@@ -15,30 +15,44 @@ end
 
 namespace :eye do
 
-  task :start, [:process] do
-
+  desc "Loads eye config and starts monitoring"
+  task :load do 
+    on release_roles fetch(:eye_roles) do
+        execute "mkdir -p #{shared_path}/config/processes"
```

Comment:
actually, i'm not sure why but eye seems like it needs that folder (I got permission errors) for temporal files